cmds/dhclient/dhclient_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/u-root/u-root/shared/testutil"
+)
+
+var tests = []struct {
+	cmd string
+	out string
+}{
+	{
+		cmd: "nosuchanimal",
```

Comment:
we've also discussed the fact that the dhcp package we're using mocks most of this stuff out anyway. Just what our tests should be is somewhat of an open question.